### PR TITLE
Shant/debugger tests

### PR DESCRIPTION
**Pull request by algonautshant**
Adding debugger coverage test.

### CODE REVIEW

File: data/transactions/logic/debugger_test.go [hunk 1]
```diff
@@ -125,3 +125,28 @@ func TestDebuggerHook(t *testing.T) {
 	require.Greater(t, testDbg.update, 1)
 	require.Equal(t, 1, len(testDbg.state.Stack))
 }
+
+func TestLineToPC(t *testing.T) {
```
Comment (tsachiherman) [inline data/transactions/logic/debugger_test.go:129]: Add missing test case for LineToPC(`negative value`) and a high value integer, i.e. 0x7fffffff

Comment (algonautshant) [reply data/transactions/logic/debugger_test.go:131]: Added.

**Review by algorandskiy — DISMISSED** [review]

**Review by tsachiherman — APPROVED** [review]
